packer/plugin.go

```diff
@@ -263,11 +300,10 @@ func (c *PluginConfig) discoverSingle(glob string) (map[string]string, error) {
 
 		// Look for foo-bar-baz. The plugin name is "baz"
 		pluginName := file[len(prefix):]
+		// For multi-compent plugins installed via the plugins split name  baz_vx.y.z_x5.0_os_arch.
+		// The plugin name is "baz"
+		pluginName = strings.SplitN(pluginName, "_", 2)[0]
```

Comment:
Given that SplitN will check for the separator and return a slice of 1 containing just the pluginName if the separator is not found, we can safely remove the conditional check.